contrib/test/integration/test.yml

```diff
@@ -7,10 +7,13 @@
     regexp: ' go test \"\$'
     state: present
 
-- name: set extra storage options
+- name: Usernamespace integration testing uses updated environment variables
   set_fact:
-    extra_storage_opts: " --storage-opt overlay.override_kernel_check=1"
-  when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
+    # Keys/Values in userns_int_test_env update int_test_env (from vars.yml)
+    int_test_env: '{{ int_test_env | combine(userns_int_test_env) }}'
+  tags:
+    - integration
+    - userns
```

Comment:
Hrmmm....I'd prefer to do this all inside ``vars.yml``, but making that aware of tag presence/absence isn't possible :confused: